mungegithub/mungers/submit-queue.go

```diff
@@ -258,10 +258,11 @@ type SubmitQueue struct {
 
 	features *features.Features
 
-	mergeLock   sync.Mutex // acquired when attempting to merge a specific PR
-	BatchURL    string
-	ContextURL  string
-	batchStatus submitQueueBatchStatus
+	mergeLock    sync.Mutex // acquired when attempting to merge a specific PR
+	ProwDataURL  string
```

Comment:
just call it ProwURL, and comment that it's the JSON jobs result page.

(Data means almost nothing without more context)